File: RxUI/src/main/java/com/tamsiree/rxui/fragment/FragmentPlaceholder.java
package com.tamsiree.rxui.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.tamsiree.rxui.R;


public class FragmentPlaceholder extends FragmentLazy {


    public static FragmentPlaceholder newInstance() {
        return new FragmentPlaceholder();
    }

    @Override
    protected View initViews(LayoutInflater layoutInflater, ViewGroup viewGroup, Bundle savedInstanceState) {
        View view = layoutInflater.inflate(R.layout.fragment_placeholder, viewGroup, false);
        return view;
    }

    @Override
    protected void initData() {

    }

}

File: RxUI/src/main/java/com/tamsiree/rxui/fragment/FragmentPlaceholder.kt
package com.tamsiree.rxui.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.tamsiree.rxui.R

class FragmentPlaceholder : FragmentLazy() {
    override fun initViews(layoutInflater: LayoutInflater, viewGroup: ViewGroup?, savedInstanceState: Bundle?): View {
        return layoutInflater.inflate(R.layout.fragment_placeholder, viewGroup, false)
    }

    override fun initData() {}

    companion object {
        fun newInstance(): FragmentPlaceholder {
            return FragmentPlaceholder()
        }
    }
}